src/client/src/components/SortableSelectionList/index.tsx

```diff
@@ -105,6 +108,7 @@ const SortableList = SortableContainer(
                   page={page}
                   handleInputChange={handleInputChange}
                   handleCloseClick={handleCloseClick}
+                  totalPageNumber = {pages.length}
```

Comment:
Very small question will mark this as resolved since it isn't a required changed but wondering why you check the pages.length inside the map function every time rather than just create something like ``` const pageLength =  pages.length ``` then use that inside the map? May just be a design decision and that's totally fine.

Reply:
Turns out I cannot mark it as resolved since I cannot request changes 😞. All good though this keeps visibility to the question.